docs/components/choice-tiles/index.html

```diff
@@ -54,14 +50,14 @@ <h3>Basic Choice Tile</h3>
             :checked="isChecked"
             :disabled="isDisabled"
             :invalid="isInvalid" />
-          <hx-tile :class="tileClasses">
+          <div class="hxTile" :class="tileClasses">
```

Comment:
Check to see if `class="hxTile"` needed.

Reply:
not needed, removing